.travis.yml

```diff
@@ -22,6 +18,6 @@ deploy:
   on:
     tags: true
     python: '3.5'
```

Comment:
Above you define that as a number, I am not sure if travis matches them. Maybe it would be better to change it here for the 3.5 number